docs/pages/guides/monorepos.mdx

```diff
@@ -107,7 +107,7 @@ Metro has three separate stages in its bundling process, [documented here](https
 
 When using monorepos, your app dependencies splits up into different directories. Each of these directories must be within the scope of the [watchFolders](https://facebook.github.io/metro/docs/configuration/#watchfolders). If a file is outside of that scope, Metro won't be able to find it. Setting this path to the root of your monorepo
 
-As your monorepo increases in size, watching all files within the monorepo becomes slower. If you want to speed that up, you can (dynamically) watch the relevant paths to minimize the watch scope. For example:
+As your monorepo increases in size, watching all files within the monorepo becomes slower. You can speed things up by only watching the packages your app uses. Typically, these are the ones that are installed with an asterik (*) in your package.json. For example:
```

Comment:
```suggestion
As your monorepo increases in size, watching all files within the monorepo becomes slower. You can speed things up by only watching the packages your app uses. Typically, these are the ones that are installed with an asterisk (*) in your **package.json**. For example:
```

Hi Peter 👋 

Thanks for this PR!

A small typo and making the [file name bold as per our writing style guide instruction](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text).

Reply:
Going to commit this suggestion so the PR do not become stale.